tests/importability/import_driver.py

```diff
@@ -90,9 +110,14 @@ def import_driver(driver_name: str) -> None:
     msg = "This script must be called with the driver name as first argument."
     raise IndexError(msg) from e
 
-if is_compatible(driver_name):
-    import_driver(driver_name)
-else:
+
+if not is_compatible(driver_name):
     logging.debug(
         f"Skipped importing {driver_name} because it is not meant to be compatible with this python version.",
     )
+elif should_skip_driver(driver_name):
+    logging.debug(
+        f"Skipped importing {driver_name} because it cannot be tested.",
+    )
```

Comment:
That message is somewhat superfluous, as the `should_skip_driver()` function already logs that information including the reason.

Reply:
Removed additional log